raw_description/urdf/schmaltz_gripper/gripper.urdf.xacro

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find cob_description)/urdf/common.xacro"/>
+  <xacro:include filename="$(find raw_description)/urdf/schmaltz_gripper/gripper.gazebo.xacro" />
+
+  <xacro:property name="gripper_length" value="0.22"/>
+  <xacro:property name="gripper_dia" value="0.07"/>
+  <xacro:property name="gripper_collision_value" value="0.01"/>
+
+  <xacro:macro name="vacuum_gripper" params="name parent *origin">
+
+    <!-- joint between arm_ee_link and vacuum_gripper_link -->
+    <joint name="${name}_base_joint" type="fixed">
+        <xacro:insert_block name="origin"/>
+        <parent link="${parent}"/>
+        <child link="${name}_frame"/>
+    </joint>
+
+    <link name="${name}_frame"/>
+
+    <!-- couplink joint between ee_link and gripper -->
+    <joint name="${name}_coupling_joint" type="fixed">
+        <parent link="${name}_frame"/>
+        <child link="${name}_coupling_link"/>
+        <origin xyz=" 0.0 0.0 0.0" rpy="0 1.57079632679 0"/>
+    </joint>
+
+    <link name="${name}_coupling_link">
+       <inertial>
+       <!--choose randomly-->
+        <origin xyz="0 0 0" rpy="0.0 0 0.0"/>
+        <mass value="0.0057307"/>
+        <inertia ixx="0.819" ixy="0" ixz="0" iyy="0.819" iyz="0" izz="0.819" />
+      </inertial>
```

Comment:
where did you get these values from?
if you don't have real inertia, it's better to approximate inertia with `box_inertia`....